examples/map/map_contouring.py

```diff
@@ -17,11 +18,19 @@
 aiamap = sunpy.map.Map(AIA_193_IMAGE)
 
 ###############################################################################
-# Now find the contours
-contours = aiamap.contour(50000)
+# In finding a set of contours, we have to provide the level to contour in the
```

Comment:
do you think it's worth explicitly mentioning here that the level here is different to level given for `draw_contours`? I feel like users may be confused slightly, and just to explain the difference is using the input `level`?

Reply:
Agreed...

Reply:
Do we need both methods?